src/connectors/clear-refinements/__tests__/connectClearRefinements-test.js

```diff
@@ -517,5 +517,26 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/clear-refin
         }
       });
     });
+
+    it('reset the page to 0', () => {
+      const helper = jsHelper({}, '', {});
+      helper.search = () => {};
+      helper.setQuery('not empty');
+
+      const rendering = jest.fn();
+      const makeWidget = connectClearRefinements(rendering);
+      const widget = makeWidget({});
+
+      widget.init({
+        helper,
+        state: helper.state,
+        createURL: () => '#',
+      });
+      const clearRefinements = rendering.mock.calls[0][0].refine;
+
+      helper.setPage(2);
+      clearRefinements();
+      expect(helper.getPage()).toBe(0);
```

Comment:
Prefer the usage of `helper.state.page`. We never know we might remove those getters at some point.

Reply:
43f9c39